dashboard/http_server_head.py

```diff
@@ -105,7 +105,12 @@ async def run(self, modules):
         app = aiohttp.web.Application(client_max_size=100 * 1024 ** 2)
         app.add_routes(routes=routes.bound_routes())
 
-        self.runner = aiohttp.web.AppRunner(app)
+        self.runner = aiohttp.web.AppRunner(
+            app,
+            access_log_format=(
+                "%a %t '%r' %s %b bytes %D us " "'%{Referer}i' '%{User-Agent}i'"
```

Comment:
This is unrelated to this PR< but I just added here because it is useful. This will log the server handler latency (%D == server handler latency)